cuda/solver/common_trs_kernels.cuh

```diff
@@ -59,16 +59,13 @@ namespace solver {
 
 
 struct SolveStruct {
-    virtual void dummy() {}
+    virtual ~SolveStruct() {}
 };
```

Comment:
might not need this anymore?

Reply:
Right, I think in this case we don't even need a vtable, but I wanted to use a more canonical way to make sure we still emit one.